contracts/native/version-control/src/queries.rs

```diff
@@ -523,24 +510,26 @@ mod test {
     use cosmwasm_std::from_binary;
 
     /// Add namespaces
-    fn add_namespaces(deps: DepsMut, namespaces: Vec<String>, account_id: u32, sender: &str) {
-        let msg = ExecuteMsg::ClaimNamespaces {
-            account_id,
-            namespaces,
-        };
+    fn add_namespaces(mut deps: DepsMut, acc_and_namespace: Vec<(u32, &str)>, sender: &str) {
```

Comment:
what ? Even if it's in tests, why are you doing that this way ? This is confusing

Reply:
Talking about the `acc_and_namespace` variable

Reply:
Ah to have the ability to register multiple namespaces anyway ?

Reply:
In case one user owns multiple accounts we can claim multiple namespaces in one. 

Fixing these tests was a huge pain btw, so some tests might be redundant at this point.

Reply:
ok, if it suits you